ttt-online/src/models/game.py

```diff
@@ -118,3 +123,14 @@ def to_dict(self) -> dict:
             if not res[prefix]['id']:
                 res[prefix] = None
         return res
+
+
+class Games(list):
+    def from_record(self, rows: Record) -> Games:
+        games = [dict(row) for row in rows]
```

Comment:
games.extends([Game.from_dict(dict(row)) for row in rows])

And i guess, that is will be good to imlement `from_record` method